common/common.scss

```diff
@@ -34,6 +35,25 @@ html:has(body.kanban-active) {
       display: flex;
     }
   }
+
+  // No sidebar. Make container fill the viewport width and center lists
+  body:not(.has-sidebar-page) &:not(.kanban-fullscreen) {
+    width: 100vw;
+    left: calc(-50vw + 50%);
```

Comment:
Doesn't this equal to zero?

Reply:
Nope. 50vw is half the width of the viewport. 50% is half the width of the containing element (in this case, `#main-outlet`).

So what we're saying here is: adjust the `left` value so that the element is positioned at the left edge of the viewport.

![SCR-20231222-lgrc-2](https://github.com/discourse/discourse-kanban-theme/assets/6270921/8a82b430-a6de-4a81-802c-461443c01088)